### PR TITLE
Fix bazel image postsubmit

### PR DESCRIPTION
**Pull request by fejta**
* docker tag can only tag one image at a time
* just tag GIT-CONFIG and latest-GIT-CONFIG

/assign @BenTheElder @chases2

### CODE REVIEW

File: images/bazel/cloudbuild.yaml [hunk 1]
```diff
@@ -2,23 +2,23 @@ steps:
...
     - build
-    - --tag=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_NEW_VERSION-from-$_OLD_VERSION
+    - --tag=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_GIT_TAG-$_CONFIG
     - --build-arg=NEW_VERSION=$_NEW_VERSION
     - --build-arg=OLD_VERSION=$_OLD_VERSION
-    - --build-arg=IMAGE_ARG=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_NEW_VERSION-from-$_OLD_VERSION
+    - --build-arg=IMAGE_ARG=gcr.io/$PROJECT_ID/launcher.gcr.io/google/bazel:$_GIT_TAG-$_CONFIG
     - .
+
   - name: gcr.io/cloud-builders/docker
```
Comment (BenTheElder) [inline images/bazel/cloudbuild.yaml:11]: Could probably override entry point to a shell and run N tag commands?

Comment (fejta) [reply images/bazel/cloudbuild.yaml:11]: Not actually sure if cloudbuild allows that? But nifty idea. Let me investigate later.

Comment (BenTheElder) [reply images/bazel/cloudbuild.yaml:11]: It does -- 
https://cloud.google.com/cloud-build/docs/build-config#entrypoint

**Pull request merged**